testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -267,7 +267,13 @@ def set_deny_with(
         headers: dict[str, ABCValue] = None,
         body: ABCValue = None,
     ):
-        """Set default deny code, message, headers, and body for 'unauthenticated' and 'unauthorized' error."""
+        """
+        Set default deny `code`, `message`, `headers`, and `body` for 'unauthenticated' and 'unauthorized' error.
+        `code` is integer range [300, 600)
+        """
+        if code:
+            assert 300 <= code < 600, "HTTP code should be in range [300, 600) for deny action."
```

Comment:
where is this specified?

Reply:
In here: 
https://pkg.go.dev/github.com/kuadrant/authorino/api/v1beta2?utm_source=gopls#DenyWithCode

Reply:
ok then, but I am not sure we need to duplicate validation. I would omit it here, it will be rejected by the OpenShift itself